pkg/selinux/selinux_test.go

```diff
@@ -18,66 +19,80 @@
 package selinux_test
 
 import (
+	"fmt"
+	"io/ioutil"
 	"os"
 	"testing"
 
 	"github.com/coreos/rkt/pkg/selinux"
 )
 
-func testSetfilecon(t *testing.T) {
-	if selinux.SelinuxEnabled() {
-		tmp := "selinux_test"
-		out, _ := os.OpenFile(tmp, os.O_WRONLY, 0)
-		out.Close()
-		err := selinux.Setfilecon(tmp, "system_u:object_r:bin_t:s0")
-		if err != nil {
-			t.Log("Setfilecon failed")
-			t.Fatal(err)
-		}
-		os.Remove(tmp)
+func TestSetfilecon(t *testing.T) {
+	if !selinux.SelinuxEnabled() {
+		t.Skip("SELinux not enabled")
+	}
+
+	f, err := ioutil.TempFile("", "rkt-selinux-test")
+	if err != nil {
+		panic(fmt.Sprintf("unable to create tempfile: %v", err))
+	}
+	f.Close()
+	defer os.Remove(f.Name())
+	err = selinux.Setfilecon(f.Name(), "system_u:object_r:bin_t:s0")
+	if err != nil {
+		t.Log("Setfilecon failed")
+		t.Fatal(err)
 	}
 }
 
 func TestSELinux(t *testing.T) {
-	var (
-		err            error
-		plabel, flabel string
-	)
+	if !selinux.SelinuxEnabled() {
+		t.Skip("SELinux not enabled")
+	}
+
+	dir, err := ioutil.TempDir("", "rkt-selinux-test")
+	if err != nil {
+		panic(fmt.Sprintf("unable to create temp dir: %v", err))
+	}
+	defer os.RemoveAll(dir)
+	_ = selinux.SetMCSDir(dir)
```

Comment:
We just created the directory and the only error that `SetMCSDir()` calls `MkdirAll()`, I can check it anyway, I guess it doesn't hurt.